modules/xmpp/JingleSessionPC.js

```diff
@@ -503,25 +503,48 @@ export default class JingleSessionPC extends JingleSession {
             // IS_SELECTED_CHANGED events and notify the peer connection
             this.rtc.addListener(RTCEvents.IS_SELECTED_CHANGED,
                 isSelected => {
-                    this.peerconnection.setIsSelected(isSelected);
-                    logger.info('Doing local O/A due to '
-                        + 'IS_SELECTED_CHANGED event');
-                    this.modificationQueue.push(finishedCallback => {
-                        if (!this._assertNotEnded('setIsSelected O/A')) {
-                            finishedCallback();
-
-                            return;
-                        }
+                    this._executeOnTheQueue('setIsSelected', finishedCallback => {
+                        logger.info('Doing local O/A due to IS_SELECTED_CHANGED event');
+                        this.peerconnection.setIsSelected(isSelected);
```

Comment:
this used to not be done as part of the queue task but now it is, is that intentional?

Reply:
I changed it thinking all peerconnection operations should be done on the queue.
Doing `setIsSelected` can be put into mid-execution of another tasks and I think it's bad.

Reply:
that may be true, just wondering if we want to change it as part of this change since it's already a bit complex (potentially).